R/tt_compatibility.R

```diff
@@ -374,6 +374,21 @@ rbindl_rtables <- function(x, gap = 0, check_headers = TRUE) {
 
     lapply(x, function(xi) chk_compat_cinfos(x[[1]], xi)) ##col_info(xi)))
 
+    rbind_annot <- list(title = "", subtitles = character(), footer = character(), pf = character())
+    # If rbinding 2 tables, annotations are retained if only present in one table or shared by both
+    if (!any(sapply(x, obj_name) == "rbind_root") & all(sapply(x, \(x) is(x, "ElementaryTable")))) {
```

Comment:
No reason they would need to be Elementary tables here.

Also, why not rbind_root, all that means is that the table came from a previous rbind, doesn't speak to annotations at all

Finally, please avoid `\(x)`. I know its valid in modern versions of R but `function(x)` isn't that much longer and is easier to read and consistent with the codebase overall.

Reply:
Updated/removed accordingly.